### PR TITLE
[ Bug #1350970] Disable non-error output from docker containers in local development

### PR DESCRIPTION
**Pull request by andela-aolasode**
## What does this PR do?

Change and add log level for docker containers

## Description of tasks to be completed

1. Add log level for container balrog ui and balrog db
2. Change log level for container balrog admin, balrog pub, balrog agent from DEBUG to WARNING

### CODE REVIEW

**Comment by njirap** [comment]
Nitpick: To me your branch name feels unnecessarily verbose.. I believe the bug number would be sufficient instead of an entire sentence. E.g something like `bg_1350970` in this case.

File: docker-compose.yml [hunk 1]
```diff
@@ -71,6 +71,7 @@ balrogui:
     - .cache:/cache
   environment:
     - WEB_PORT=8080
+    - LOG_LEVEL=WARNING
```
Comment (mozbhearsum) [inline docker-compose.yml]: I don't think this will do anything for the ui and db containers. Remove this and then it's good to go!

Comment (andela-aolasode) [inline docker-compose.yml]: I have implemented the changes. The log level for ui and db containers have been removed.

**Comment by mozbhearsum** [comment]
Looks good, thank you very much!

**Pull request merged**